net.mkiol.SpeechNote.yaml

```diff
@@ -0,0 +1,428 @@
+          md5: c9924cdb6305f6982171d486d5d633e1
+        - type: patch
+          path: patches/piper.patch
+
+    - name: pcre2
+      buildsystem: cmake
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+        - -DPCRE2_STATIC_PIC=ON
+        - -DBUILD_STATIC_LIBS=ON
+        - -DBUILD_SHARED_LIBS=OFF
+        - -DPCRE2_SUPPORT_LIBBZ2=OFF
+        - -DPCRE2_SUPPORT_LIBZ=OFF
+        - -DPCRE2_SUPPORT_JIT=ON
+      sources:
+        - type: archive
+          url: https://github.com/PCRE2Project/pcre2/releases/download/pcre2-10.42/pcre2-10.42.tar.bz2
+          md5: a8e9ab2935d428a4807461f183034abe
+
+    - name: ssplitcpp
+      buildsystem: cmake
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+        - -DSSPLIT_COMPILE_LIBRARY_ONLY=ON
+        - -DSSPLIT_PREFER_STATIC_COMPILE=ON
+        - -DBUILD_SHARED_LIBS=OFF
+      sources:
+        - type: archive
+          url: https://github.com/ugermann/ssplit-cpp/archive/49a8e12f11945fac82581cf056560965dcb641e6.zip
+          md5: 8255bd212ad639592b4e212b0353509a
+        - type: patch
+          path: patches/ssplitcpp.patch
+
+    - name: pybind11
+      buildsystem: cmake
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+        - -DBUILD_SHARED_LIBS=OFF
+        - -DPYBIND11_INSTALL=ON
+        - -DPYBIND11_TEST=OFF
+        - -DPYBIND11_FINDPYTHON=ON
+      sources:
+        - type: archive
+          url: https://github.com/pybind/pybind11/archive/refs/tags/v2.10.4.tar.gz
+          md5: 812eda11d2a114fc0e841faf9626d2c9
+
+    - name: rnnoise
+      buildsystem: autotools
+      config-opts:
+        - --disable-examples
+        - --disable-doc
+        - --disable-shared
+        - --enable-static
+        - --with-pic
+      sources:
+        - type: archive
+          url: https://github.com/GregorR/rnnoise-nu/archive/26269304e120499485438cd93acf5127c6908c68.zip
+          md5: fafe9bbf0e2b15df4a628434bea99dd3
+
+    - name: rhvoice
+      buildsystem: cmake
+      config-opts:
+        - -DCMAKE_BUILD_TYPE=Release
+        - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+        - -DWITH_DATA=OFF
+        - -DENABLE_MAGE=OFF
+        - -DBUILD_CLIENT=OFF
+        - -DBUILD_UTILS=OFF
+        - -DBUILD_TESTS=OFF
+        - -DBUILD_SERVICE=OFF
+        - -DBUILD_SPEECHDISPATCHER_MODULE=OFF
+        - -DCMAKE_INSTALL_LIBDIR=/app/lib
+      post-install:
+        - 'mkdir -p /app/share/rhvoice'
+        - 'cp -R data /app/share/rhvoice'
+        - 'cp -R config /app/share/rhvoice'
+      sources:
+        - type: git
+          url: https://github.com/RHVoice/RHVoice.git
+          commit: c1e897d90c663613086398cd019882e907d02314
+        - type: patch
+          path: patches/rhvoice.patch
+
+    - python3-modules-x86-64.yaml
+
+    - name: dsnote-x86_64
```

Comment:
same as above, you can make it one module.

Reply:
Fixed in 8df0e78